Build/README.md

```diff
@@ -0,0 +1,97 @@
+# Continuous Integration for Newtonsoft.Json-for-Unity.Converters
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[maximum-heading-length] Use headings shorter than `60`](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5459451)